pkg/executioncluster/execution_target.go

```diff
@@ -11,7 +10,11 @@ import (
 // Spec to determine the execution target
 type ExecutionTargetSpec struct {
 	TargetID    string
-	ExecutionID *core.WorkflowExecutionIdentifier
+	ExecutionID string
```

Comment:
why not keep the ExecutionID to make it backwards compatible and add the Workflow Identifier + Launch Plan Identifier as additional params? the project+domain aren't always identical across execution, workflow and launch plan anyways, no?

Reply:
Ohhhhh interesting. I guess will then use the launch plan's value.

Reply:
@katrogan I thought more. This is actually a weird state for resource overrides. 

I will use the Domain, project from the execution. And name of launchplan and workflow.
What that would mean is Execution launched in a different project/domain would need a separate override even if the workflow does not exist there.

Reply:
ok that sounds reasonable. how are you planning on migrating the db? (we can talk about this separately from the PR offline)